### ISSUE TITLE
Connections panel scrolling down when focussed

### ISSUE THREAD

**Issue by rzuidhof** [mRemoteNG/mRemoteNG#1925]
Sometimes my Connections panel keeps scrolling down when it has focus from the mouse pointer. Clicking or selecting with arrow keys is still possible but with a long list a groups the top ones will disappear from view fast.

Using mRemoteNG 1.76.20 on Windows Server 2012 R2. I think it also happened with previous releases.

Frequency is hard to tell, maybe once every two months? Restarting the application fixes the issue.

**Comment by ehubgit** [comment]
This is the same issue in Windows 10/11 as well. Observed it very often.
Running mRemoteNG stable v1.76.20

**Comment by simonai1254** [comment]
Do I understand it correctly, that the connection panel scrolls even though the mouse wheel is not touched? 
It is wanted behavior that it scrolls when the mouse hovers the list and the scroll wheel is moved, but as soon as the wheel is stopped the scrolling should stop as well...

**Comment by ehubgit** [comment]
Yes it scrolls even when the mouse wheel is not touched. When mouse hovers on connections list without the wheel scrolling.

**Comment by antonostrovsky** [comment]
Same here: 1.76.20.24615. But I've found a workaround: collapse all your folders (if any) and change the order of elements in the connection tree. The scrolling stops

**Comment by robertpopa22** [comment]
Triage refresh (P4 version-label debt): this issue is tagged with legacy version labels. Please retest on the latest build (fork v1.79.x or upstream dev), then share reproducible steps and expected vs actual behavior. If still valid, keep open and relabel to current triage scope (for example Need 2 check). If fixed, close with resolution.

Current legacy version labels: 1.77.2

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
